src/components/ConfirmTermination/ConfirmTerminationStyles.ts

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { transparentize } from 'polished';
+import { themeColor } from '@amsterdam/asc-ui';
+
+export interface Props {
+	backdropOpacity?: number;
+	zIndexOffset?: number;
+}
+
+export const BackDropStyle = styled.div<Props>`
```

Comment:
I suppose we weren't able to re-purpose the backdrop rendered from ASC?

Reply:
No, the backdrop would fall behind all rendered Modals